client/src/featureform/get.py

```diff
@@ -58,154 +39,118 @@ def get_resource_info(stub, resource_type, name):
-            format_rows([("NAME: ", x.name),
-            ("VARIANT: ", x.variant),
-            ("OWNER:", x.owner),
-            ("DESCRIPTION:", x.description),
-            ("PROVIDER:", x.provider),
-            ("STATUS: ", x.status.Status._enum_type.values[x.status.status].name)])
-            format_pg("LABEL: ")
-            format_rows([("NAME", "VARIANT"), (x.label.name, x.label.variant)])
-            format_pg("FEATURES:")
-            format_rows("NAME", "VARIANT")
-            for f in x.features:
-                format_rows(f.name, f.variant)
-            format_pg()
-            return x
+            return ff.TrainingSet(
+                name=x.name,
+                variant=x.variant,
+                label=(x.label.name,x.label.variant),
+                status=x.status.Status._enum_type.values[x.status.status].name,
+                description=x.description,
+                owner=x.owner,
+                schedule=x.schedule,
+                features=[(f.name,f.variant) for f in x.features],
+                feature_lags=None,
+            )
     except grpc._channel._MultiThreadedRendezvous:
         print("Training set variant not found.")
 
 def get_provider_info(stub, name):
     searchName = metadata_pb2.Name(name=name)
     try:
         for x in stub.GetProviders(iter([searchName])):
-            format_rows([("NAME: ", x.name),
-            ("DESCRIPTION: ", x.description),
-            ("TYPE: ", x.type),
-            ("SOFTWARE: ", x.software),
-            ("TEAM: ", x.team),
-            ("STATUS: ", x.status.Status._enum_type.values[x.status.status].name)])
-            format_pg("SOURCES:")
-            format_rows("NAME", "VARIANT")
-            for s in x.sources:
-                format_rows(s.name, s.variant)
-            format_pg("FEATURES:")
-            format_rows("NAME", "VARIANT")
-            for f in x.features:
-                format_rows(f.name, f.variant)
-            format_pg("LABELS:")
-            format_rows("NAME", "VARIANT")
-            for l in x.labels:
-                format_rows(l.name, l.variant)
-            format_pg("TRAINING SETS:")
-            format_rows("NAME", "VARIANT")
-            for t in x.trainingsets:
-                format_rows(t.name, t.variant)
-            format_pg()
-            return x
+            return ff.Provider(
+                name=x.name,
+                description=x.description,
+                team=x.team
+                software=x.software,
+                provider_type=x.type,
+                status=x.status.Status._enum_type.values[x.status.status].name,
+                sources=[(f.name,f.variant) for f in x.sources],
+                features=[(f.name,f.variant) for f in x.features],
+                trainingsets=[(f.name,f.variant) for f in x.trainingsets],
+                labels=[(f.name,f.variant) for f in x.labels],
+                config=ff.SerializedConfig(serialized=x.serialized_config),
```

Comment:
Should leave this out. Don't know of a reason why we would pass this back to a user right now